packages/cloud-functions/uma-tvl-calculator/index.js

```diff
@@ -0,0 +1,27 @@
+const { Datastore } = require("@google-cloud/datastore");
+const datastore = new Datastore();
+
+const { calculateCurrentTvl } = require("@uma/merkle-distributor");
+
+exports.calculateCurrentTvl = async (req, res) => {
```

Comment:
In the GCP configuration, is this one going to be publicly accessible too? Or have you configured it s.t. this one is only accessible via cloud scheduler, but the accessors are public?

Reply:
this is not accessible publicly. it's called directly from a scheduler on a recurring basis (every 5 mins right now)

Reply:
+1 -- thanks for the explanation. Makes sense!